feat(smartcar): Get & Set Charge Limit

EvanPeterson1324 opened the pull request:
## Overview
Adds one new data type `VehicleChargeLimit` with two new methods, `getChargeLimit()` and `setChargeLimit(double limit)`

Asana: https://app.asana.com/0/1204073873904879/1204302264096976/f

### Checklist
- [x] Get Charge Limit & javadoc
- [x] Set Charge Limit & javadoc
- [x] Unit tests
- [x] Integration tests

s-ashwinkumar commented:
@EvanPeterson1324 The tests are working now. I dont know how this changed but there was no redirect URI in the configs . That said, there is still something you might have to look into in code.
This is the current failure logs : https://app.travis-ci.com/github/smartcar/java-sdk/jobs/599949031
If you scroll down to find the issue you will see this 

```
> Task :compileTestJava FAILED
/home/travis/build/smartcar/java-sdk/src/test/java/com/smartcar/sdk/VehicleTest.java:223: error: unexpected type
    Assert.assertTrue(chargeLimit.getChargeLimit() instanceof Double);
                                                ^
  required: reference
  found:    double
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
1 error
```

Points to a code issue that we introduced.

EvanPeterson1324 commented:
> @EvanPeterson1324 The tests are working now. I dont know how this changed but there was no redirect URI in the configs . That said, there is still something you might have to look into in code. This is the current failure logs : https://app.travis-ci.com/github/smartcar/java-sdk/jobs/599949031 If you scroll down to find the issue you will see this
> 
> ```
> > Task :compileTestJava FAILED
> /home/travis/build/smartcar/java-sdk/src/test/java/com/smartcar/sdk/VehicleTest.java:223: error: unexpected type
>     Assert.assertTrue(chargeLimit.getChargeLimit() instanceof Double);
>                                                 ^
>   required: reference
>   found:    double
> Note: Some input files use or override a deprecated API.
> Note: Recompile with -Xlint:deprecation for details.
> 1 error
> ```
> 
> Points to a code issue that we introduced.

I fixed the above issue :+1: I needed to wrap the primitive `double` with the `Double` constructor for it to check properly with `instanceof`

EvanPeterson1324 commented on src/main/java/com/smartcar/sdk/data/VehicleChargeLimit.java:
```diff
@@ -0,0 +1,25 @@
+        + limit
+        + '}';
+  }
+}
```
```suggestion
}

```

s-ashwinkumar reviewed (changes requested):
Code looks OK to me but you also need to bump the versions. In java sdk it is manually done in the grade properties and readme i think.
Reference PR from last week : https://github.com/smartcar/java-sdk/pull/129/files#diff-3d103fc7c312a3e136f88e81cef592424b8af2464c468116545c4d22d6edcf19L3
